code/API_definitions/number-recycling.yaml

```diff
@@ -244,11 +295,24 @@ components:
       content:
         application/json:
           schema:
-            $ref: '#/components/schemas/ErrorInfo'
-          example:
-            status: 404
-            code: NOT_FOUND
-            message: A specified resource is not found
+            allOf:
+              - $ref: "#/components/schemas/ErrorInfo"
+              - type: object
+                properties:
+                  status:
+                    enum:
+                      - 404
+                  code:
+                    enum:
+                      - IDENTIFIER_NOT_FOUND
+          examples:
+            GENERIC_404_IDENTIFIER_NOT_FOUND:
+              description: Provided phone number is not one used by network operator
+              value:
+                status: 404
+                code: IDENTIFIER_NOT_FOUND
+                message: Phone number not found.
+
```

Comment:
```suggestion
```

Reply:
404 IDENTIFIER_NOT_FOUND is not applicable to this API. In this case, the API returns `true `in  `phoneNumChanged` parameter.

Reply:
That's not the purpose of this error. The error is used when, due to a routing error, Operator A receives a number recycling request for a MSISDN that is actually used by Operator B. That's a clear error, because Operator A can say nothing about the MSISDN, so they return a 404 IDENTIFIER_NOT_FOUND error. The API consumer then knows they need to ask a different API provider about that MSISDN.